core/domain/draft_upgrade_services.py

```diff
@@ -87,6 +87,80 @@ def try_upgrading_draft_to_exp_version(
     return draft_change_list
 
 
+def extract_html_from_draft_change_list(draft_change_list):
```

Comment:
I don't think this is a duplicate.because this new method just checks the type of draft changes and extracts HTML accordingly, but the method _convert_states_v33_dict_to_v34_dict converts HTML.
Both the methods need to do check the type of draft changes and proceed accordingly.
even the method exp_services.apply_change_list uses the similar code